tools/bench/core/numerics/interpolator/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required (VERSION 3.9)
```

Comment:
Some edits in this PR have `(VERSION 3.20.1)`, is that consistent?

Reply:
https://github.com/PHAREHUB/PHARE/blob/master/CMakeLists.txt#L1 

cmake was complaining about old versions